feature/gribi/otg_tests/ipv4_entry_with_aggregate_ports_test/README.md

```diff
@@ -17,26 +18,34 @@ Validate IPv4 support in gRIBI using an Aggregate Port as a static route Next Ho
 
     *   Single IPv4Entry -> NHG -> NH with MAC Override.
 
-        *   Install 198.51.100.0/24 to NextHopGroup containing one NextHop which
-            is a static route to the ATE LAG port containing ports 2-3, and
-            override the destination MAC to a specified value.
-
-        *   Forward packets between ATE port-1 and ATE LAG (destined to
-            198.51.100.0/24 i.e. packets with destination IP starting
-            198.51.100.1 up to 198.51.100.255) and determine that packets are
-            forwarded successfully.
-
-        *   Disable ATE port-2 and forward packets between ATE port-1 and ATE
-            LAG (destined to 198.51.100.0/24 ) and determine that packets are
-            forwarded successfully.
-
-        *   Disable ATE port-2 and port-3 and forward packets between ATE port-1
-            and ATE LAG (destined to 198.51.100.0/24 ) and determine that
-            packets are lost 100%.
-
-        *   Re-enable both ATE port-2 and port-3 and forward packets between ATE
-            port-1 and ATE LAG (destined to 198.51.100.0/24 ) and determine that
-            packets are forwarded successfully again.
+        *   Configure a static ARP entry for the LAG interface pointing the
```

Comment:
Looks like this part is written based on a specific vendor implementation (currently gated by a deviation)

Should this really be a part of the README? This wasn't the approach used in TE-2.1, and not a requirement AFAIK.

We also have PR #2415 that tries to rectify the current version of the test and readme with TE-2.1.

cc @xw-g

Reply:
This is a requirement for some vendors that don't support MAC address override through gRIBI.

However, the scenario itself is very common in our prod environment and we expect to have similar static routes pointing to Aggregate Interfaces in prod.

Reply:
The PR got merged, but lets discuss further. @xw-g Any comments?

Reply:
> This is a requirement for some vendors that don't support MAC address override through gRIBI.

Yep, but that's a "workaround" / how they implement it.
But the base scenario is a direct gribi NH/NHG/route combo with a mac override that goes into a README, isn't it?

> we expect to have similar static routes pointing to Aggregate Interfaces in prod.

We believe the static route isn't required either, if you can specify an aggregate interface directly as a gribi NH? Just like TE-2.1 uses Ethernet interfaces directly as a NH (and the static route is a deviation/workaround)

Reply:
So we want to validate such a scenario.

Reply:
What is the purpose of the static route? Is there any other use of it beyond the recursive resolution of gribi routes?

Reply:
I am not aware of the specific purpose behid the static routes. @xw-g will have more context.

Reply:
Ok, let's wait for feedback from Xiao.

As far as we understand it, the base scenario is that gribi NH specifies an interface + mac directly (in this particular test case, an aggregate interface + mac).

All additional constructs (static routes, magic IP addresses, static ARPs) arise from a simple fact that not all implementations can do the above directly.

Reply:
This part (static route + static ARP) for archiving the MAC overriden should be gated by a deviation (as other test did). An example:

https://github.com/openconfig/featureprofiles/blob/5d59959528e125fee6639b716c2a30322e6e99da/internal/deviations/deviations.go#L129

https://github.com/openconfig/featureprofiles/blob/5d59959528e125fee6639b716c2a30322e6e99da/feature/gribi/otg_tests/base_hierarchical_nhg_update/base_hierarchical_nhg_update_test.go#L273

Reply:
HI @xw-g the `MAC override` is not relevant to the test. I think we can remove the MAC override part from the test since we don't need it. The only thing we need is gRIBI route --> StaticRoute --> LAG Interface

Reply:
>  The only thing we need is gRIBI route --> StaticRoute --> LAG Interface

Or, to make it simpler, `gRIBI route --> LAG Interface`? :-)

I understand where are you coming from (the issue you described above), but again, in our view the usage of the static route itself is a deviation (with or without MAC override) and the base scenario (for a fully gribi-compliant implementation) is a plain gRIBI interface NH (or interface+MAC nh)

Reply:
Hi @LimeHat Had a discussion with @xw-g and you are right. We don't need the static route for Nokia. 

Will review https://github.com/openconfig/featureprofiles/pull/2415